vulcan/lib/client/jsx/components/workflow/user_interactions/inputs/boolean.tsx

```diff
@@ -16,10 +16,10 @@ export default function BooleanInput({onChange, label, data, ...props}: WithInpu
     />
 
   if (label) {
-    return <label className='checkbox-input-option'>
+    return <div className='checkbox-input-option' style={{display: 'inline-flex'}}>
```

Comment:
Made this `label` to `div` change in order to be able to encourage the checkbox and label to be in one line.